src/scib_metrics/utils/_utils.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+
+from .._types import NdArray
+
+
+def get_ndarray(x: jnp.ndarray) -> np.ndarray:
+    """Convert Jax device array to Numpy."""
+    return np.array(jax.device_get(x))
+
+
+def one_hot(y: NdArray, n_classes: Optional[int] = None) -> NdArray:
```

Comment:
have you looked at https://jax.readthedocs.io/en/latest/_autosummary/jax.nn.one_hot.html?

Reply:
Not sure how I missed that, thanks!